### DISCUSSION TITLE
InfluxDB notification endpoints

### DISCUSSION THREAD

**Discussion by rosswakelin** [Security-Onion-Solutions/securityonion#11136] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I'm starting down the process of creating a replacement for the Grafana alerting, around such things as node down, monitor interface traffic levels etc.  Looking at the InfluxDB screens, I can see a number of supplied checks (thank you), but the only notification endpoints available are the Influx built-in ones of Slack, http and pagerduty.  Is there any support for adding additional endpoints e.g. email, ServiceNow, Teams?

Thanks

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
InfluxDB's Flux language allows users to create custom notifications. For example, email can be sent via third-party services like AWS, MailGun, SendGrid, etc. Example:
https://docs.influxdata.com/influxdb/v2.7/monitor-alert/send-email/?t=AWS+SES

There are also user-contributed Flux modules for utilizing other services like Alerta, ServiceNow, Teams. See the Standard Library -> Contrib section of the Flux documentation:
https://docs.influxdata.com/flux/v0.x/
